Mathlib/Algebra/Order/Sub/Basic.lean

```diff
@@ -206,7 +206,7 @@ namespace CanonicallyOrderedAdd
 
 variable [AddCommMonoid α] [LinearOrder α] [CanonicallyOrderedAdd α]
 
--- See note [reducible non instances]
+-- See note [reducible noninstances]
```

Comment:
This note (and similar occurrences) will need to change, if you revert the above